### ISSUE TITLE
NaN Loss during the training in DTU dataset

### ISSUE THREAD

**Issue by ewrfcas** [FangjinhuaWang/PatchmatchNet#53]
![image](https://user-images.githubusercontent.com/12042259/138989704-0544d66a-944c-47fa-ac59-a75de9a0446e.png)
Hello, I run the default codes and settings in DTU dataset. The loss becomes NaN with about 5k steps.

My pytorch version 1.3.1

Is the preporcessed data different from MVSNet? I use the processed data from MVSNet directly.
